userspace/libsinsp/sinsp.cpp

```diff
@@ -1202,8 +1202,9 @@ int32_t sinsp::next(OUT sinsp_evt **puevt)
 		{
 			if (m_external_event_processor)
 			{
-				m_external_event_processor->process_event(NULL, libsinsp::EVENT_RETURN_EOF);
+				m_external_event_processor->process_event(evt, libsinsp::EVENT_RETURN_EOF);
```

Comment:
#1423 didn't touch these lines? I see that this code is there since ~3yrs at least; a quick blame showed that: https://github.com/falcosecurity/libs/blame/682a4a9ea917dabff8df75bd739ee446a940251e/userspace/libsinsp/sinsp.cpp

Reply:
You're right. That's not needed.